api/external/applications/applications.proto

```diff
@@ -94,7 +94,7 @@ message ServicesReq {
 }
 
 message ServicesBySGReq {
-  int32 service_group_id = 1;
+  string service_group_id = 1;
```

Comment:
this is usually very very bad but we're beta so I'd rather do this than carry the cruft forever